Tools/CISupport/ews-build/steps.py

```diff
@@ -2150,6 +2174,188 @@ def hideStepIf(self, results, step):
         return not self.doStepIf(step)
 
 
+class FilterPullRequestsIntoQueues(buildstep.BuildStep, GitHubMixin, AddToLogMixin):
```

Comment:
This name seems too generic and the name doesn't give much idea about what this class/step is excepted to do.

Can rename it to something like: `CheckStatusOfPRsInSafeMergeQueue` (and corresponding step name as well).